.github/workflows/continuous-integration.yml

```
@@ -34,20 +38,26 @@
     strategy:
       fail-fast: false
       matrix:
-        php-version: ${{ fromJson(inputs.old_stable) }}
+        # If inputs.old_stable is not an empty string and not null (i.e., a valid value came from workflow_call or workflow_dispatch), use it.
+        # The default: '[]' for inputs.old_stable applies if it's omitted in workflow_call.
+        # For push/pull_request, inputs.old_stable will be an empty string, so env.DEFAULT_OLD_STABLE_JSON is used.
+        php-version: ${{ fromJson(inputs.old_stable || env.DEFAULT_OLD_STABLE_JSON) }}
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)